keps/sig-node/2535-ensure-secret-pulled-images/kep.yaml

```diff
@@ -18,11 +18,11 @@ approvers:
 prr-approvers:
   - "@johnbelarmic"
 stage: alpha
-latest-milestone: "v1.24"
+latest-milestone: "v1.26"
 milestone:
-  alpha: "v1.24"
-  beta: "v1.25"
-  stable: "v1.27"
+  alpha: "v1.26"
```

Comment:
Version needs an update. I will follow up the PR ASAP.

Reply:
@mikebrow do you have time to update the KEP?

Probably, v1.28 for alpha.

Reply:
done